common/include/pcl/impl/instantiate.hpp

```diff
@@ -35,14 +35,8 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  *
  */
-#ifndef PCL_IMPL_INSTANTIATE_H_
-#define PCL_IMPL_INSTANTIATE_H_
 
-#ifdef __GNUC__
-#pragma GCC system_header 
-#endif
-
-#include <pcl/pcl_config.h>
```

Comment:
You removed this include and thus you don't get `PCL_NO_PRECOMPILE` defined.

It's a very happy coincidence we caught this problem in tutorials CI. IWUY is a treacherous tool.

Reply:
That's a real sad thing. I dislike macros, and now I think I hate macros without values. Thanks for the catch

Reply:
Looks like it worked. Nice magic @taketwo